### PR TITLE
Finish task generate event from controller (case repeat daily)

### PR DESCRIPTION
**Pull request by tqtrung09**

### CODE REVIEW

**Comment by vietbq** [comment]
ok

File: app/controllers/api/events_controller.rb [hunk 1]
```diff
@@ -16,7 +16,8 @@ def index
       end
     else
       @events = Event.in_calendars params[:calendars]
-      @data = @events.map{|event| event.json_data(current_user.id)}
+      @data = GenerateEventFullcalendarServices.new(@events)
```
Comment (dieunb) [inline app/controllers/api/events_controller.rb]: `@data = GenerateEventFullcalendarServices.new(@events).data_repeat_of_user current_user`

File: app/services/generate_event_fullcalendar_services.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+class GenerateEventFullcalendarServices
+  attr_accessor :events
+
+  def initialize events
+    @events = events
```
Comment (dieunb) [inline app/services/generate_event_fullcalendar_services.rb]: @user = user

File: lib/models/event_fullcalendar.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+class EventFullcalendar
+  include SharedMethods
+
+  attr_accessor :id, :title, :description, :status, :color, :all_day,
```
Comment (dieunb) [inline lib/models/event_fullcalendar.rb]:
```
ATTRS = [:id, :title, :body]
attr_accessor *ATTRS
```

File: lib/models/event_fullcalendar.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    :repeat_type, :repeat_every, :user_id, :calendar_id, :start_date, :finish_date,
+    :start_repeat, :end_repeat, :exception_time, :exception_type, :event_id
+
+  def initialize event
+    @id, @title, @description, @status, @color, @all_day, @repeat_type,
```
Comment (dieunb) [inline lib/models/event_fullcalendar.rb]:
```
ATTRS[1..-1].each do |attr|
  instance_variable_set "@#{attr}", event.send(:attr)
end
```

**Pull request merged**